### ISSUE TITLE
An editor is being restored when I use workbench.action.focusLeftGroup in empty group

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#50693]
VS Code version: Code - Insiders 1.24.0-insider (41a7c4976ebd1f6ced20c034485300f46ab1829f, 2018-05-29T06:09:34.618Z)
OS version: Windows_NT x64 10.0.16299

---

#50458

Starting state (right editor is focused):

![image](https://user-images.githubusercontent.com/2193314/40672103-a66c18a6-6322-11e8-94c3-66945308a347.png)

Trigger this keybinding:

```json
{ "key": "alt+left",        "command": "workbench.action.focusLeftGroup", "when": "editorTextFocus" },
```

Results in:

![image](https://user-images.githubusercontent.com/2193314/40672135-bdf3d1b2-6322-11e8-9d4b-ce720e005995.png)

**Comment by bpasero** [comment]
@Tyriar I cannot reproduce and by looking at the implementation of the action ([here](https://github.com/Microsoft/vscode/blob/ben/editor/src/vs/workbench/browser/parts/editor/editorActions.ts#L320)) I see no code that would ever open an editor. 

Can you reproduce reliably? Could it be that some other command got executed because you have a keybinding collision?

**Comment by Tyriar** [comment]
@bpasero I don't think it would be a keybinding conflict, when I was in this state it would happen about 75% of the time when moving left.

Cannot repro now on a different machine, I'm fine if you close this as as not reproducible as nothing looks wrong from the code to me either, it was definitely happening though :slightly_smiling_face:
